### PR TITLE
[Lab] Add button to disable devmode and Chucker toggle.

### PR DESCRIPTION
**Pull request by Antoni-Czaplicki**

### CODE REVIEW

File: app/src/main/java/pl/szczodrzynski/edziennik/App.kt [hunk 1]
```diff
@@ -58,6 +58,7 @@ class App : MultiDexApplication(), Configuration.Provider, CoroutineScope {
         val profileId
             get() = profile.id
 
+        var chucker = false
```
Comment (kuba2k2) [inline app/src/main/java/pl/szczodrzynski/edziennik/App.kt:61]: Change this to `enableChucker` here and in Config.

File: app/src/main/java/pl/szczodrzynski/edziennik/App.kt [hunk 2]
```diff
@@ -172,6 +175,7 @@ class App : MultiDexApplication(), Configuration.Provider, CoroutineScope {
         App.profile = Profile(0, 0, 0, "")
         debugMode = BuildConfig.DEBUG
         devMode = config.debugMode || debugMode
+        chucker = config.chucker
```
Comment (kuba2k2) [inline app/src/main/java/pl/szczodrzynski/edziennik/App.kt:178]: Enable by default when `devMode` is enabled.

**Review by kuba2k2 — CHANGES_REQUESTED** [review]

File: app/src/main/java/pl/szczodrzynski/edziennik/App.kt (hunk 1 above)
Comment (Antoni-Czaplicki) [reply app/src/main/java/pl/szczodrzynski/edziennik/App.kt]: done

File: app/src/main/java/pl/szczodrzynski/edziennik/App.kt (hunk 2 above)
Comment (Antoni-Czaplicki) [reply app/src/main/java/pl/szczodrzynski/edziennik/App.kt]: done

File: app/src/main/java/pl/szczodrzynski/edziennik/App.kt [hunk 3]
```diff
@@ -172,6 +175,10 @@ class App : MultiDexApplication(), Configuration.Provider, CoroutineScope {
         App.profile = Profile(0, 0, 0, "")
         debugMode = BuildConfig.DEBUG
         devMode = config.debugMode || debugMode
+        if (devMode)
+            enableChucker = true
+        else
+            enableChucker = config.enableChucker
```
Comment (kuba2k2) [inline app/src/main/java/pl/szczodrzynski/edziennik/App.kt:181]: `enableChucker = config.enableChucker || devMode`

**Review by kuba2k2 — CHANGES_REQUESTED** [review]

**Review by kuba2k2 — APPROVED** [review]

**Pull request merged**